packages/fx-core/resource/package.nls.json

```diff
@@ -246,6 +246,11 @@
   "core.copilotPlugin.validate.openAIPluginManifest.summary": "Teams Toolkit has checked your OpenAI plugin manifest:\n\nSummary:\n%s.\n%s\n%s",
   "core.copilotPlugin.validate.summary.validate.failed": "%s failed",
   "core.copilotPlugin.validate.summary.validate.warning": "%s warning",
+  "core.copilotPlugin.scaffold.summary": "We have detected the following issues for your OpenAPI Specification file:\n%s",
+  "core.copilotPlugin.scaffold.summary.warning.operationId": "%s Mitigation: Not required, operationId has been automatically generated and added into %s file.",
```

Comment:
```"%s Mitigation: Not required```
You should add "." after %s

Reply:
I am expecting you will return a complete sentence ending with "."  :)

Reply:
Sure, maybe I can change it in my next pr